clients/algoliasearch-client-scala/src/main/scala/algoliasearch/config/Host.scala

```diff
@@ -8,5 +8,7 @@ package algoliasearch.config
   *   Whether this host should be used for read and/or write requests.
   * @param scheme
   *   Host protocol (i.e. `http`, `https`)
+  * @param port
+  *   Host port
   */
-case class Host(url: String, callTypes: Set[CallType], scheme: String = "https")
+case class Host(url: String, callTypes: Set[CallType], scheme: String = "https", port : Option[Int] = None)
```

Comment:
Sad the linter didn't catch it 😬 Not a mandatory change right now, though (can be updated in another PR)

```suggestion
case class Host(url: String, callTypes: Set[CallType], scheme: String = "https", port: Option[Int] = None)
```

Reply:
the linter catches it in fact, the generated code will have the fix, it's just not pushed becaused of PHP 😠

Reply:
Perfect